pmd-cataloger/src/main/java/sfdc/sfdx/scanner/messaging/MessageHandler.java

```diff
@@ -0,0 +1,6 @@
+package sfdc.sfdx.scanner.messaging;
+
+public enum MessageHandler {
```

Comment:
Why does this need to be in its own file? And it looks like it's not used anywhere outside of the `messaging` package, so does it need to be `public`? Can't it just be default-scoped?